docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -162,7 +154,7 @@ A deploy alias consists of:
     
       This tutorial uses `berkeley`, but the deploy alias name can be anything and does not have to match the network name.
   
-    - Target network: `testnet`
+    - Target network kind: `Testnet`
     
     - Mina GraphQL API URL: `https://proxy.berkeley.minaexplorer.com/graphql`
```

Comment:
I think we should use `minascan` and deploy to `devnet` here.

Reply:
Will be done as part of the work on https://github.com/o1-labs/docs2/issues/916.